source/NVDAObjects/window/winword.py

```diff
@@ -1187,7 +1187,7 @@ def winwordColorToNVDAColor(self,val):
 			return colors.RGB.fromCOLORREF(val).name
 		elif (val&0xffffffff)==0xff000000:
 			# Translators: the default (automatic) color in Microsoft Word
-			return _("default color")
+			return _("automatic color")

```

Comment:
I'm not sure if this change is appropriate, can you add a comment explaining why the specific term is chosen?

Reply:
e.g. matches reporting in other case

Reply:
Hi Cyrille, I was suggesting to add a comment to the "automatic" strings noted that they must be kept consistent with each other. Alternatively, factor them out to a named constant

Reply:
Sorry, I do not understand.
Do you mean that there are two strings with the same content, i.e. "automatic color"?
Where is the other one?

Reply:
Never mind, my assumption was there were other instances in the UX of "automatic color" being reported